File: CapitalesAfrica_Java/app/src/main/java/com/example/africajava/Inicio.java
package com.example.africajava;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class Inicio extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_inicio);
        InicializaBD bdd = new InicializaBD(this); //creamos la bd
        Button btnJugar = findViewById(R.id.btnJugar);
        Button btnSalir = findViewById(R.id.btnSalir);
        btnJugar.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                bdd.resetearBd(); //limpiamos la bd
                bdd.insertaPaises(); //insertamos los países
                Intent i = new Intent(Inicio.this, MainActivity.class); //creamos la nueva actividad
                startActivity(i); //arrancamos la nueva actividad
            }
        });
        btnSalir.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                finishAffinity(); //cerramos la app
            }
        });
    }
}

File: CapitalesAfrica_Kotlin/app/src/main/java/com/example/africakotlin/Inicio.kt
package com.example.africakotlin

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.ImageButton

class Inicio : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_inicio)
        val bdd = InicializaBD(this) //arrancamos la bd
        val btnJugar = findViewById<Button>(R.id.btnJugar)
        val btnSalir = findViewById<Button>(R.id.btnSalir)
        btnJugar.setOnClickListener {
            bdd.resetearBd() //limpiamos la bd
            bdd.insertaPaises() //insertamos los países
            val i = Intent(this@Inicio, MainActivity::class.java) //creamos la nueva actividad
            startActivity(i) //iniciamos la nueva actividad
        }
        btnSalir.setOnClickListener {
            finishAffinity() //cerramos la app
        }
    }
}